bin/fixmystreet.com/one-off-status-update

```diff
@@ -78,13 +78,26 @@ if (!$commit) {
     say "*** DRY RUN ***";
 }
 
-my %alloy_to_fms_map = (
+my %enq_to_fms_map = (
+    'Issued to Inspector' => 'investigating',
+    'In Progress' => 'investigating',
```

Comment:
not "in progress"?

Reply:
No, because in progress in this case means an inspection is in progress, rather than fixing the problem is in progress.